install/helm/README.md

```diff
@@ -20,18 +20,45 @@ This repository contains the Helm chart for WSO2 Thunder, a lightweight user and
 
 Follow these steps to deploy Thunder in your Kubernetes cluster:
 
-### 1. Clone the Thunder repository
+### 1. Install the Thunder Helm chart
+
+You have multiple options to install the Thunder Helm chart:
+
+#### Option 1: Install from OCI Registry (GitHub Container Registry) - **Recommended**
+
+This is the recommended installation method as it provides:
+- Faster installation (no need to clone the repository)
+- Version-controlled releases
+- Better security and reliability
```

Comment:
We don't need these justifications if this is the only option as mentioned in https://github.com/asgardeo/thunder/pull/720/files#r2515292015